client/config.go

```diff
@@ -71,6 +73,18 @@ func (config *Config) RemoveRemote(name string) bool {
 	}
 	return false
 }
+func (config *Config) SetProperty(name string, value string) bool {
+	f := reflect.ValueOf(config).Elem().FieldByName(name)
+	if f.IsValid() {
```

Comment:
From the `reflect` documentation:

> IsValid reports whether v represents a value. It returns false if v is the zero Value.

Does this mean that if the config currently has no value set for a field, you won't be able to set it? I think if the field is `null` we should still allow users to set a value to it.

Reply:
no IsValid() is checking whether a field(key) is found is the struct. The implementation of that is checking a flag resides in the value struct.

Reply:
ohh I see